server/redux/reducers/room-reducer.js

```diff
@@ -0,0 +1,59 @@
+const { Map } = require('immutable');
+
+/* --------------- INITIAL STATE --------------- */
+
+const initialState = Map({});
+
+/* --------------- ACTIONS --------------- */
+
+const ADD_ROOM = 'ADD_ROOM';
+const ADD_SOCKET_TO_ROOM = 'ADD_SOCKET_TO_ROOM';
+const REMOVE_SOCKET_TO_ROOM = 'REMOVE_SOCKET_TO_ROOM';
```

Comment:
sort of semantically confusing wording. I see you're using removeSocketFromRoom below. Can you change this to match?